core-tests/jvm/src/test/scala/zio/FiberRefSpecJvm.scala

```diff
@@ -0,0 +1,45 @@
+package zio
+
+import java.util.concurrent.Executors
+
+import zio.internal.Executor
+
+import scala.concurrent.ExecutionContext
+
+class FiberRefSpecJvm extends BaseCrossPlatformSpec {
+  override def is = "FiberRefSpec".title ^ s2"""
```

Comment:
I put the executor related tests into a separate, JVM only spec, because I had trouble to make this compile with ScalaJS (they don't have `java.util.concurrent.Executors`). If there is a better way, let me know.